### PR TITLE
Add Chinese translation

### PR DESCRIPTION
**Pull request by moechakucha**
Complete Chinese translations.

### CODE REVIEW

File: src/main/resources/assets/yttr/lang/zh_cn.json
Comment (Neubulae) [inline src/main/resources/assets/yttr/lang/zh_cn.json]: This references Yttr-ium as a separate element from Yttrium, would it be fine to do so as it introduces inconsistencies with gadolinite, plus Yttrium is just… Yttrium?
上文翻译是硅铍钇矿石，这里却是伊特尔了…这种不一致真的好吗（
伊特尔元素应当就是钇元素就是。

File: src/main/resources/assets/yttr/lang/zh_cn.json
Comment (Neubulae) [inline src/main/resources/assets/yttr/lang/zh_cn.json:65]: 虚空玻璃/虚空琉璃会不会好听点？

File: src/main/resources/assets/yttr/lang/zh_cn.json
Comment (Neubulae) [inline src/main/resources/assets/yttr/lang/zh_cn.json:139]: oof, why is this one a different line?
怎么刀尖上跳舞就变了呢？

File: src/main/resources/assets/yttr/lang/zh_cn.json
Comment (Neubulae) [inline src/main/resources/assets/yttr/lang/zh_cn.json:141]: This is supposed to be a process of denaturization/void-devoiring instead of… protein denature?
单纯理解为蛋白质变性似乎不是很贴切的样子。

File: src/main/resources/assets/yttr/lang/zh_cn.json
Comment (Neubulae) [inline src/main/resources/assets/yttr/lang/zh_cn.json:174]: Oof.

File: src/main/resources/assets/yttr/lang/zh_cn.json
Comment (Neubulae) [inline src/main/resources/assets/yttr/lang/zh_cn.json]: "自食苦果"

File: src/main/resources/assets/yttr/lang/zh_cn.json
Comment (Neubulae) [reply src/main/resources/assets/yttr/lang/zh_cn.json:139]: "玩家在<物品名称>的刀尖上跳舞"应该好点吧。

File: src/main/resources/assets/yttr/lang/zh_cn.json
Comment (Neubulae) [inline src/main/resources/assets/yttr/lang/zh_cn.json:162]: Is Food Value referencing Hunger Restored?
"饱食度"应该好点吧，因为wiki上的说法确实是饱食饱和分开的。

File: src/main/resources/assets/yttr/lang/zh_cn.json
Comment (Neubulae) [inline src/main/resources/assets/yttr/lang/zh_cn.json:94]: It was "分离" in RedPower 2's locale.

**Review by Neubulae — CHANGES_REQUESTED** [review]

File: src/main/resources/assets/yttr/lang/zh_cn.json
Comment (exaskye) [reply src/main/resources/assets/yttr/lang/zh_cn.json:174]: I don't know if a rarely-used administrative command needs to be localized. Would be nice if it were, though.

File: src/main/resources/assets/yttr/lang/zh_cn.json
Comment (exaskye) [reply src/main/resources/assets/yttr/lang/zh_cn.json:162]: In the original English, yes, "Food Value" effectively means "Hunger Restored". I'm not sure why I went with that wording.

File: src/main/resources/assets/yttr/lang/zh_cn.json
Comment (exaskye) [reply src/main/resources/assets/yttr/lang/zh_cn.json:94]: Disjunction is a direct reference to RP2, so it would be nice to have the localizations of it match where reasonable to keep the reference clear. I didn't even know RP2 had localizations...

File: src/main/resources/assets/yttr/lang/zh_cn.json
Comment (exaskye) [reply src/main/resources/assets/yttr/lang/zh_cn.json:141]: In English this is a somewhat-intentionally awkward turn of phrase suggested by someone else to emphasize the "alien-ness" of the Void and that it's "against the planet's wishes" or similar to use Void Diving as travel.

**Comment by exaskye** [comment]
Thanks for the review, Neubulae. I was just going to run it through Google Translate to make sure it didn't have a bunch of bad words. :P

File: src/main/resources/assets/yttr/lang/zh_cn.json
Comment (Neubulae) [reply src/main/resources/assets/yttr/lang/zh_cn.json:174]: It often needs to be tho, because people may use administrative commands to their wishes xd

**Comment by Neubulae** [comment]
> Thanks for the review, Neubulae. I was just going to run it through Google Translate to make sure it didn't have a bunch of bad words. :P

I reviewed it earlier that day and found out I have to explicitly click "Start A Review" to post my comments...
Overall this translation is quite good in terms of quality other than the issues I have mentioned, it should be good to go after they address the issues.

File: src/main/resources/assets/yttr/lang/zh_cn.json
Comment (ChromaPIE) [inline src/main/resources/assets/yttr/lang/zh_cn.json:187]: In vanilla we call Pink in color series "粉红色", you got a character missing.

File: src/main/resources/assets/yttr/lang/zh_cn.json
Comment (ChromaPIE) [inline src/main/resources/assets/yttr/lang/zh_cn.json:56]: Stem is not "原木", it's "菌柄". Vanilla term.

File: src/main/resources/assets/yttr/lang/zh_cn.json
Comment (ChromaPIE) [inline src/main/resources/assets/yttr/lang/zh_cn.json:91]: This kind of status actaully makes players **less** depend on consuming edible items, so it makes zero sense translating "Delicaceness", a malformed word coming from a Google Translate meme, to "食欲旺盛" (Hyperphagia).

File: src/main/resources/assets/yttr/lang/zh_cn.json
Comment (Neubulae) [reply src/main/resources/assets/yttr/lang/zh_cn.json:56]: 实际上这应该是个木头？

File: src/main/resources/assets/yttr/lang/zh_cn.json
Comment (Neubulae) [reply src/main/resources/assets/yttr/lang/zh_cn.json:91]: "精致感" "感觉良好" maybe?

File: src/main/resources/assets/yttr/lang/zh_cn.json
Comment (Neubulae) [reply src/main/resources/assets/yttr/lang/zh_cn.json:141]: “<玩家>受到了星球的排异反应”(Player triggered the planet's (immunologic) rejection) sounds about it then?

**Comment by moechakucha** [comment]
God, I've had SO much fun(maybe) editing the file...

**Comment by Neubulae** [comment]
This is probably in quite a good shape I mean. Can already be merged?

**Comment by exaskye** [comment]
I suppose; I was under the impression there were still unresolved issues, and I'm a bit confused by the removal of translations of Yttrium, Neodymium, and Gadolinite. These are real-world materials, and Yttrium/Neodymium have extant translations.

I don't speak Chinese, though, so my opinion on this really has no meaning; maintainer or not.

**Comment by exaskye** [comment]
Thanks for making the initial translation, Peakstep233, and thanks Neubulae and ChromaPIE for assisting with review.

**Comment by Neubulae** [comment]
> I suppose; I was under the impression there were still unresolved issues, and I'm a bit confused by the removal of translations of Yttrium, Neodymium, and Gadolinite. These are real-world materials, and Yttrium/Neodymium have extant translations.
> 
> I don't speak Chinese, though, so my opinion on this really has no meaning; maintainer or not.

I have commented them about it but it's probably OK for the time being. They probably thought about them being non-existent ores, only the name being references.

**Pull request merged**
